chapters/it/_toctree.yml

```diff
@@ -0,0 +1,4 @@
+- title: 0. Installazione
+  sections:
+  - local: chapter0/1
+    title: Introduzione
```

Comment:
I think we lost the piece for `chapter1/1` 😬 . Could you add the following below this line:

```yaml
- title: 1. Modelli Transformer
  sections:
  - local: chapter1/1
    title: Introduzione
```

Reply:
Thanks for this! It seems that we just have some merge conflicts to resolve on the `build_pr_documentation.yml` and `build_documentation.yml` files - would you be happy to fix that?

Reply:
Ok so I've just resolved my first conflicts ever. Hopefully I didn't make a mess!